lib/dest/options.js

```diff
@@ -29,6 +33,18 @@ var config = {
     type: ['string', 'boolean'],
     default: false,
   },
+  // Symlink options
+  useJunctions: {
+    type: 'boolean',
+    default: function(file) {
+      var isDirectory = file.isDirectory();
+      return (isWindows && isDirectory);
```

Comment:
Maybe the default should just be a boolean value, set to `isWindows`? We have to AND the resolved option with `isDirectory(file)` per file anyway, because we probably shouldn't assume that the `useJunctions` option provided by the user always satisfies that restriction. Does that make sense?